<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTILBOM20102124" failures="0" tests="1" name="com.automation.selenium_template.AllureTests" time="16.403" errors="1" timestamp="2023-09-28T13:36:18 IST" skipped="0">
  <testcase classname="com.automation.selenium_template.AllureTests" name="dc" time="16.403">
    <error message="Cannot throw exception because the return value of &quot;java.lang.Throwable.getCause()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot throw exception because the return value of "java.lang.Throwable.getCause()" is null
at com.automation.selenium_template.reports.AllureReportingProvider.lambda$2(AllureReportingProvider.java:50)
at io.qameta.allure.Allure.lambda$step$1(Allure.java:127)
at io.qameta.allure.Allure.step(Allure.java:181)
at io.qameta.allure.Allure.step(Allure.java:125)
at com.automation.selenium_template.reports.AllureReportingProvider.reportStepFailure(AllureReportingProvider.java:48)
at com.automation.selenium_template.reports.ReportUtil.reportStepFailure(ReportUtil.java:43)
at com.automation.selenium_template.driver.DriverControllerV5.reportStepFailure(DriverControllerV5.java:2862)
at com.automation.selenium_template.driver.DriverControllerV5.waitForVisibilityOfElement(DriverControllerV5.java:1603)
at com.automation.selenium_template.driver.DriverControllerV5.waitForVisibilityOfElement(DriverControllerV5.java:1580)
at com.automation.selenium_template.driver.DriverControllerV5.sendKeys(DriverControllerV5.java:398)
at com.automation.selenium_template.AllureTests.dc(AllureTests.java:61)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- dc -->
</testsuite> <!-- com.automation.selenium_template.AllureTests -->
